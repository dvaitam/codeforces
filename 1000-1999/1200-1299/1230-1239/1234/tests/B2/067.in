26 4
19 13 16 18 20 8 9 2 6 18 17 8 14 9 14 13 9 16 4 5 6 18 1 15 2 16
