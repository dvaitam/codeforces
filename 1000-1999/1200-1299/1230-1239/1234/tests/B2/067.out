4
16 2 15 1
